freud/diffraction.py

```diff
@@ -836,12 +849,15 @@ def to_image(self, cmap="afmhot", vmin=None, vmax=None):
             ((output_size, output_size, 4) :class:`numpy.ndarray`):
                 RGBA array of pixels.
         """
-        if not _HAS_MPL:
-            raise ImportError(msg_mpl)
```

Comment:
To appease the pre-commit and the linter you want to revert the changes on the conditional imports to the code that is current with the main branch.